src/v2/cookbook/serverless-blog.md

```diff
@@ -0,0 +1,289 @@
+---
+title: Create a CMS-Powered Blog
+type: cookbook
+order: 5
+---
+
+# Create a serverless CMS-Powered Blog Using Vue.js
+
+So you've just launched your Vue.js website, congrats! Now you want to add a blog that quickly plugs into your website and you don't want to have to spin up a whole server just to host a Wordpress instance (or any DB-powered CMS for that matter). You want to just be able to add a few Vue.js blog components and some routes and have it all just work, right? What you're looking for a "serverless" blog that's powered entirely by API's you can consume directly from your Vue.js application. This tutorial will teach you how to do just that, let's dive in!
+
+We're going to quickly build a serverless CMS-powered blog with Vue.js. It uses [ButterCMS](https://buttercms.com/), an API-first CMS that lets you manage content using the ButterCMS dashboard and integrate our content API into your Vue.js app. You can use ButterCMS for new or existing Vue.js projects.
+ 
+![Butter Dashboard](https://user-images.githubusercontent.com/160873/36677285-648798e4-1ad3-11e8-9454-d22fca8280b7.png "Butter Dashboard")
+
+## Install
+
+Run this in your commandline:
+
+`npm install buttercms --save`
+
+Butter can also be loaded using a CDN: 
+
+`<script src="https://cdnjs.buttercms.com/buttercms-1.1.0.min.js"></script>`
+
+## Quickstart
+
+Set your API token:
+
+`var butter = require('buttercms')('your_api_token');`
+
+Using ES6:
+
+```javascript
+import Butter from 'buttercms';
+const butter = Butter('your_api_token');
+```
+
+Using CDN: 
+
+```javascript
+<script src="https://cdnjs.buttercms.com/buttercms-1.1.0.min.js"></script>
+<script>
+  var butter = Butter('your_api_token');
+</script>
+```
+ 
+ Import this file into any component you want to use ButterCMS. Then from the console run:
+ 
+```javascript
+butter.post.list({page: 1, page_size: 10}).then(function(response) {
+  console.log(response)
+})
+```
+
+This API request fetches your blog posts. Your account comes with one example post which you'll see in the response.
+
+## Display posts
+To display posts we create a `/blog` route (using vue-router) in our app and fetch blog posts from the Butter API, as well as a `/blog/:slug` route to handle individual posts. See our [API reference](https://buttercms.com/docs/api/?javascript#blog-posts) for additional options such as filtering by category or author. The response also includes some metadata we'll use for pagination.
+
+See our API reference for additional options such as filtering by category or author. The response also includes some metadata we'll use for pagination.
```

Comment:
In this section, you say "our API reference" a few times- please say "the ButterCMS API reference" as you're in the vue docs and "our API reference" should refer to Vue's API reference.

Reply:
👍